docs/resources/job.md

```diff
@@ -207,6 +207,13 @@ This block controls notification settings for both email & webhook notifications
 * `no_alert_for_skipped_runs` - (Optional) (Bool) don't send alert for skipped runs.
 * `no_alert_for_canceled_runs` - (Optional) (Bool) don't send alert for cancelled runs.
 
+###  job_parameters Configuration Block
```

Comment:
We use singular names for the blocks that represent one or more entities (e.g., the `task` block).

```suggestion
###  job_parameter Configuration Block
```